.gitlab-ci.yml

```diff
@@ -149,12 +146,19 @@ before_script:
     - |
       if [[ ${CI_JOB_NAME} =~ "install" ]];
       then
-        make install;
+        make install
+        touch tests.xml
       else
         ./sesame2spiner/sesame2spiner -s materials.sp5 ../sesame2spiner/examples/unit_tests/*.dat;
         ./sesame2spiner/sesame2spiner -s duplicates.sp5 ../sesame2spiner/examples/duplicate-test/*.dat;
-        make CTEST_OUTPUT_ON_FAILURE=1 test;
+        export CTEST_OUTPUT_ON_FAILURE=1
+        ctest --output-junit tests.xml
```

Comment:
just curious, did this output format get added or was it always there?

Reply:
It's been added in CMake 3.21